recipes/pyodide-lock/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "0.1.0a1" %}
+{% set version = "0.1.0a3" %}
```

Comment:
Will they release a stable version soon? Should we apply CFEP-05 here?

https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
I don't believe there is a concrete upstream timeline, just following the threads on https://github.com/conda-forge/pyodide-build-feedstock/pull/6 ...

Reply:
We have room for exceptions. Do you think this is one?

Reply:
Sure, we ended up doing so for e.g. [lsprotocol](https://pypi.org/project/lsprotocol/) as it has a lot of production downstreams. 

As I'm doing some work upstream on this, I am pretty confident I can keep it working here, but totally understand if it's not worth the exception... while non-zero, the numbers for the downstream are still [quite low](https://anaconda.org/conda-forge/pyodide-build/files), and we could leave this open and revisit if someone complains.